### PR TITLE
Get involved page

### PR DESCRIPTION
**Pull request by sjpaige**
Integrated the Get Involved subpages and worked on a possible solution for the nav cards, its a little weird I was having clashes with the css when using the grid item layout, but with just the flex container it works.

### CODE REVIEW

File: src/Pages/GetInvolvedPage.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import NavCardGroup from "../Components/NavCardGroup";
+import DonatePage from "./DonatePage";
+import PartnerPage from "./PartnerPage";
+import VolunteerPage from "./VolunteerPage";
+
+const subPages = [
+  {
+    name: "Volunteer",
+    link: "/get_involved/volunteer",
+    desc: "Join our dedicated volunteer team and help us bring theresources to those who need them.",
```
Comment (stuttskl) [inline src/Pages/GetInvolvedPage.jsx:15]: missing space between the and resources

File: src/Pages/VolunteerPage.jsx [hunk 2]
```diff
@@ -195,7 +195,7 @@ const VolunteerPage = () => {
                 >
                   If you want further information and want to have a chat, don’t
                   hesitate to contact us by email at{" "}
-                  <a href="/contact">contact@cvp.com</a> and we’ll get back to
+                  <a href="/contact">convictionvacation@gmail.com</a> and we’ll get back to
```
Comment (stuttskl) [inline src/Pages/VolunteerPage.jsx:198]: Could we make this the yellow color in our theme? I think it'll help make it pop and the blue is a little hard to see on the blue background

**Review by stuttskl — APPROVED** [review]
Approved pending comments

File: src/Pages/GetInvolvedPage.jsx (hunk 1 above)
Comment (sjpaige) [reply src/Pages/GetInvolvedPage.jsx:15]: good catch

File: src/Pages/VolunteerPage.jsx (hunk 2 above)
Comment (sjpaige) [reply src/Pages/VolunteerPage.jsx:198]: yes! I have been thinking we need to expand that yellow into some of the other areas.

**Pull request merged**